sunpy/map/map_factory.py

```diff
@@ -243,15 +245,20 @@ def __call__(self, *args, composite=False, sequence=False, silence_errors=False,
             Indicates if collection of maps should be returned as a `sunpy.map.MapSequence`.
             Default is `False`.
         silence_errors : `bool`, optional
+            Deprecated.
```

Comment:
```suggestion
            Deprecated, renamed to `allow_errors`.
```

Reply:
or something along those line so that the user knows that they do the same thing

Reply:
Makes sense, I will add this.